tests/acceptance/features/bootstrap/WebDav.php

```diff
@@ -3354,58 +3354,51 @@ public function userDeletesFile(string $user, string $file):void {
 	}
 
 	/**
-	 * @Given /^user "([^"]*)" has (deleted|unshared) (?:file|folder|entity) "([^"]*)"$/
+	 * @Given /^user "([^"]*)" has deleted (?:file|folder|entity) "([^"]*)"$/
 	 *
 	 * @param string $user
-	 * @param string $deletedOrUnshared
-	 * @param string $entry
+	 * @param string $resource
 	 *
 	 * @return void
 	 * @throws Exception
 	 */
-	public function userHasDeletedFile(string $user, string $deletedOrUnshared, string $entry):void {
+	public function userHasDeletedFile(string $user, string $resource):void {
```

Comment:
```suggestion
	public function userHasDeletedResource(string $user, string $resource):void {
```

Reply:
There are many step definition codes like this WebDav and Sharing files where similar refactoring can be done. 
Example: the word entity can be replaced in every code by 'resources', and the regex statement also looks redundant in the code.
@saw-jan @amrita-shrestha @grgprarup @PrajwolAmatya @SwikritiT

Reply:
IMO, those can be done in other refactor PR is they are not related to unshare and delete steps